internal/cmd/kubernetes.go

```diff
@@ -85,80 +85,82 @@ func writeKubeconfig(user *api.User, destinations []api.Destination, grants []ap
 		return err
 	}
 
-	keep := make(map[string]bool)
+	type clusterContext struct {
```

Comment:
```suggestion
	type context struct {
```

Reply:
I wanted to avoid clobbering the `context` package

Reply:
Good call!